packages/amplify_storage_s3/android/src/main/kotlin/com/amazonaws/amplify/amplify_storage_s3/types/FlutterDownloadFileRequest.kt

```diff
@@ -47,15 +48,14 @@ data class FlutterDownloadFileRequest(val request: Map<String, *>) {
     }
 
     companion object {
-        fun isValid(request: Map<String, *>): Boolean {
-            var valid = true
-            if (request["path"] !is String) {
-                valid = false
+        private const val validationErrorMessage: String = "Download request malformed."
+        fun isValid(request: Map<String, *>) {
+            if(request["path"] !is String?) {
+                throw AmplifyFlutterValidationException(validationErrorMessage, "path is missing.")
```

Comment:
These shouldn't be called `isValid` anymore since they don't return anything but throw error. Maybe `validate` is a better name?

Reply:
Done.